### PR TITLE
Update governance documentation

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary
Cleared some misc. items up in the governance docs.  
  

## PR Context  
Some stuff didn't make much sense, or needed a more in-depth explanation.  

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [X] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [X] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by iSazonov — APPROVED** [review]

File: docs/community/governance.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ After the RFC has been discussed, a unanimous vote will be required for the new
 
 ## Repository Maintainers
 
-Repository Maintainers are trusted stewards of the PowerShell repository responsible for maintaining consistency and quality of PowerShell code.
+Repository Maintainers are trusted users of the PowerShell community responsible for maintaining consistency and quality of PowerShell code/repository.
```
Comment (TravisEz13) [inline docs/community/governance.md]: I don't understand the clarification here.  Changing from stewards to users is a significant change in the meaning here (triggering the committee review.)   
```suggestion
Repository Maintainers are trusted stewards of the PowerShell community/repository responsible for maintaining consistency and quality of PowerShell code.
```

I suggest this change to keep the original meaning.

File: docs/community/governance.md [hunk 2]
```diff
@@ -97,6 +97,8 @@ They have [write access](https://help.github.com/articles/permission-levels-for-
 1. Merge pull requests to all branches *except* `master` (though this should not be common given that [`master`is the only long-living branch](../git/README.md#understand-branches)).
 1. Assign labels, milestones, and people to [issues](https://guides.github.com/features/issues/).
 
+A list of Area Experts can be found [here][experts]
```
Comment (TravisEz13) [inline docs/community/governance.md]:
```suggestion
A list of Area Experts can be found [here][experts].
```

**Comment by TravisEz13** [comment]
filed #8784 and restarted macOS

**Comment by iSazonov** [comment]
@TravisEz13 Could you please merge?

**Pull request merged**